addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -868,6 +868,25 @@ export function preserveCursor(document) {
     };
 }
 
+/**
+ * Check if the selection starts inside given selector. This function can be
+ * used as the `isDisabled` property of a command of the PowerBox to disable
+ * a command in the given selectors.
+ * @param {string}: comma separated string with all the desired selectors
+ * @returns {boolean} true selector is within one of the selector
+ * (if the command should be filtered)
+ */
+export function isSelectionInSelectors(selector) {
+    let anchor = document.getSelection().anchorNode;
+    if (anchor && anchor.nodeType && anchor.nodeType !== Node.ELEMENT_NODE) {
+        anchor = anchor.parentElement;
+    }
+    if (anchor && closestElement(anchor, selector)) {
+        return true;
+    }
+    return false;
+}
```

Comment:
Unless I misunderstood something, `closestElement` already does those checks, and you're in master so you can even use the optional chaining operator, so all in all I feel like this entire function basically amounts to:
```js
return closestElement(document.getSelection()?.anchorNode, selection);
```
Not sure it's really worth a function in that case, right ? ^^'

Reply:
@dbeguin He is saying things far beyond my remaining cerebral capacities for this freeze.

Reply:
@dmo-odoo  ah indeed. I just moved the code from our side but indeed, your proposal does the job directly :)
Thanks !

Reply:
You're welcome but ... this commit is still there at r+ time so I'm not sure what is going on here... ^^'